sdk/servicebus/service-bus/src/util/utils.ts

```diff
@@ -143,7 +143,7 @@ export function toBuffer(input: any): Buffer {
     // string, undefined, null, boolean, array, object, number should end up here
     // coercing undefined to null as that will ensure that null value will be given to the
     // customer on receive.
-    if (input === undefined) input = null; // tslint:disable-line
+    if (input === undefined) input = null; // eslint:disable-curly
```

Comment:
I'd rather have this fixed than disabling the rule

Reply:
fixed